lodash/lodash.d.ts

```diff
@@ -10445,6 +10446,13 @@ declare module _ {
         isEmpty(): boolean;
     }
 
+    interface LoDashExplicitWrapperBase<T,TWrapper> {
+        /**
+         * @see _.isEmpty
+         */
+        isEmpty(): LoDashExplicitWrapper<boolean>;
```

Comment:
@DomiR thanks for your review. It is a case when the chaining used explicitly, e.g. `_('').chain().isEmpty()`. So the returning of wrapper is correct here.